packages/jam/transition/statistics.ts

```diff
@@ -232,22 +235,19 @@ export class Statistics {
     current[authorIndex].preImagesSize = tryAsU32(newPreImagesSize);
 
     /**
-     * * NOTE [MaSi] Please note I don't use Kappa' here. If I understand correctly we don't need it.
-     * Kappa' is not needed because we can use validator indexes directly from guarantees extrinsic.
-     * I asked a question to ensure it is true but I didn't get any response yet:
-     * https://github.com/w3f/jamtestvectors/pull/28#discussion_r190723700
+     * Update guarantees
      *
-     * https://graypaper.fluffylabs.dev/#/1c979cb/19a00119a801?v=0.7.1
+     * https://graypaper.fluffylabs.dev/#/ab2cdbd/19ea0119f201?v=0.7.2
      */
-    const incrementedGuarantors = new Set<ValidatorIndex>();
-    for (const guarantee of extrinsic.guarantees) {
-      for (const { validatorIndex } of guarantee.credentials) {
-        if (!incrementedGuarantors.has(validatorIndex)) {
-          const newGuaranteesCount = current[validatorIndex].guarantees + 1;
-          current[validatorIndex].guarantees = tryAsU32(newGuaranteesCount);
-          incrementedGuarantors.add(validatorIndex);
-        }
+    const validatorKeys = input.currentValidatorData.map((v) => v.ed25519);
+
+    for (const reporter of input.reporters) {
+      const index = validatorKeys.findIndex((x) => x.isEqualTo(reporter));
+      if (index === -1) {
+        continue; // Reporter not in current validator set
```

Comment:
Can you clarify whether it's an intended behaviour to ignore these in the statistics or if that should not happen since we validate that in some other place (what place?)?

Just from my intuition I feel reporters should either be in the current set or the previous one, so I guess we should have some validation someplace else and here we just don't count.